### PR TITLE
Makes the briefcase the lawyer starts with very robust, and makes it fit on the belt slot

### PR DESCRIPTION
**Pull request by JF260**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

## Changelog
:cl:
add: Makes the lawyers briefcase more robust, and it now fits on the belt slot
/:cl:

## About The Pull Request
Gives the lawyers starting briefcase 14 force (where toolboxes are 12) and allows it to fit on your belt slot.

## Why It's Good For The Game
Makes the lawyer slightly more interesting. In the future, I will make more lawyer changes for the sake of making lawyer an actual job. Here are some of my ideas:
Giving lawyers an instant summons automatically linked to their briefcase, and find some IC way to explain it or at least make it thematic.
Encourage admins to enforce contracts.
Allow them to fold paper into different things.
But hey, that's all for later. For now, this gives lawyer an iota of individuality, instead of just being the love child of sec and assistant.

### CODE REVIEW

File: code/game/objects/items/storage/briefcase.dm [hunk 1]
```diff
@@ -29,6 +29,11 @@
 
 /obj/item/storage/briefcase/lawyer
 	folder_path = /obj/item/folder/blue
+	force = 14
+	throwforce = 14
+	throwspeed =2
+	throw_range = 7
+	slot_flags = ITEM_SLOT_BELT
```
Comment (Jujumatic) [inline code/game/objects/items/storage/briefcase.dm]: Reeee modularise

**Review by Jujumatic — CHANGES_REQUESTED** [review]

File: code/game/objects/items/storage/briefcase.dm (hunk 1 above)
Comment (JF260) [reply code/game/objects/items/storage/briefcase.dm]: fair, gimme a sec

Comment (JF260) [reply code/game/objects/items/storage/briefcase.dm]: done

**Review by Jujumatic — APPROVED** [review]

**Comment by Manly-Dwarf** [comment]
Can you still knock people out by hitting them on the head?

**Comment by JF260** [comment]
> Can you still knock people out by hitting them on the head?

The thing where you knock people silly? I'm not sure, but if its possible, that would be a fun addition.
EDIT: Cant find it in the code, but im pretty sure it exists. I'll keep looking, but for now this works
EDIT: Cant find a general knockout, but I could try to splice in the boxing glove knockout if youd like

**Comment by yoyobatty** [comment]
It's under species.dm or human_defense.dm somewhere maybe spec_attackedby

**Comment by Maroonenahmoron** [comment]
I dunno about teleporting it, you can put several objects in a briefcase like traitor objectives or syndicate items and being able to summon them out of the blue is unbalanced. Otherwise, I’m happy with 14 force and being put on belt slot. +1

**Comment by Lxtoofonea** [comment]
+1 for what mr maroon said

**Comment by Jujumatic** [comment]
needs one more +1

**Comment by earthmoonjune** [comment]
sure +1

**Comment by Manly-Dwarf** [comment]
-1 for shilling the PR

**Pull request merged**